fields/types/cloudinaryimage/CloudinaryImageField.js

```diff
@@ -292,7 +295,7 @@ module.exports = Field.create({
 		if (this.state.userSelectedFile || this.state.removeExisting) {
 			const value = this.state.userSelectedFile
 				? `upload:${this.state.uploadFieldPath}`
-				: '';
+				: `upload:remove`;
```

Comment:
Good catch, but I think checking for `remove` (mirrored in `CloudinaryImageType.js` below) is probably clearer